sum.c

```diff
@@ -0,0 +1,23 @@
+/*
+Created on november 8, 2018
+@author: sumangala rao
+*/
+//program to find the sum inclusive of the start and the end no. mentioned
+  #include<stdio.h>
```

Comment:
Let these remain left aligned.
You should start indenting for every block. That is, after every flower braces.